src/api/axiosInstance.ts

```diff
@@ -0,0 +1,11 @@
+import axios from "axios";
+
+const axiosInstance = axios.create({
+  baseURL: "http://localhost:4000/api/v1",
```

Comment:
You can e.g. use something like `Number(process.env.APP_PORT) ?? 4000` here to parametrize the for the development and the build time (and easily synchronize it with the back-end, if the need arises)